web/src/routes/OffersPage.tsx

```diff
@@ -52,8 +54,16 @@ const OffersPage = ({ cityModel, cityCode, languageCode, languages }: CityRouteP
           thumbnail: offer.thumbnail,
           postData: offer.postData
         })
-      }),
-    [cityCode, languageCode]
+      })
+
+      if (cityCode === 'augsburg') {
```

Comment:
If more cities join / also want to use this offer it would be nice to have a constant somewhere, like "citiesWithWohnraum".